tests/model_serving/model_server/maas_billing/conftest.py

```
@@ -594,15 +630,278 @@ def maas_host(admin_client):
+    ):
+        yield
+
+
+@pytest.fixture(scope="session")
+def maas_gateway_api_hostname(admin_client: DynamicClient) -> str:
+    return host_from_ingress_domain(client=admin_client)
+
+
+@pytest.fixture(scope="session")
+def maas_gateway_and_policies(
+    maas_gateway_api: None,
```

Ruff (0.14.13)

Unused function argument: `maas_gateway_api`

(ARG001)